### ISSUE TITLE
Feature request: pictures in recipies

### ISSUE THREAD

**Issue by bzz** [creachadair/recipes#5]
There are some really delightful treats in the list, but it may sometimes be hard to imagine how the final result of following the cooking procedure should look like.

It would be really nice if not to taste and smell, then at least to see the results.

**Comment by creachadair** [comment]
I agree! I have photos on some of them, and there’s a place in the template for it, but I haven’t actually got photos of all of these recipes yet. I’ll add more as I have occasion to take them. 👍🏼
